pkg/config/specification_test.go

```diff
@@ -0,0 +1,19 @@
+package config
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_LoadEnv(t *testing.T) {
+	os.Setenv("PORT", "8001")
```

Comment:
`PORT`
Problem is that with current code this sets both values, and `API_PORT` must use default value.